### ISSUE TITLE
Once the character is "Created", Provide an History of the AP spent

### ISSUE THREAD

**Issue by ZeSandman** [elyukai/optolith-client#608]
History/journal of how many points was spent on what.

Right now, I use diffpdf to see the difference between two character versions, but it still mean that I have to save every "milestone" and compare them.

Having an "AP Spending history" would permit players and GM to "audit"  their stuff and review if the spending was "logical" story-wise.

I guess it could be done automatically when saving a character as Optolith has an Undo History and can always compare what was loaded to what is saved.

The order of the changes themselves aren't important, only that "XYZ was added on day ABC"

It could potentially be "printed" as a simple additional page on the PDF

And It could be tied-in with #592, the Adventure Journal

**Comment by elyukai** [comment]
This is basically #37, isn't it?

I'm not sure if a history of changes based on the date of change makes sense, though.

**Comment by ZeSandman** [comment]
Ha, yes it seems to match pretty much #37...

I wouldn't "base" it on the date, but just noting the date would help the player/gm remember WHEN it was done and why... ("Oh, yeah, right, we had talked about XYZ and now that make sense!")  a simple "timestamp" on the note would be ok.   Not every group meets every week or bi-weekly, some meet montly or less.. 

And no, not every little itty bitty changes needs to be tracked... like I mentioned, just the "Diff" between the loaded & saved version should be noted

Can you ad on the Wishlist the possibility to add a comment on the actual change?

**Comment by elyukai** [comment]
>  but just noting the date would help the player/gm remember WHEN it was done and why...
Yeah, that's certainly important, so kind of the "standard" Date, Title, and Description fields.

> And no, not every little itty bitty changes needs to be tracked... like I mentioned, just the "Diff" between the loaded & saved version should be noted
I don't know, I'd rather diff different "semantic" versions, like "After Adventure X" and "After Adventure Y" or sth like that instead of just every save. So that each savepoint has an actual meaning.

> Can you ad on the Wishlist the possibility to add a comment on the actual change?
Yes, I'm gonna do that! 👍🏻

**Comment by elyukai** [comment]
Updated! 👍🏻
